recipes/openmp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "openmp" %}
```

Comment:
So there's gcc and intel openmp, but I think they are built into the packages by default.